firestore/testapp/README.md

```diff
@@ -33,6 +33,14 @@ Building and Running the testapp
           sudo gem install cocoapods --pre
         ```
 
+    -   Update the pod versions in the Podfile to match the C++ SDK version that you are using.
```

Comment:
Would probably be useful to mention where to look this up. How about adding a sentence like:
```
For the latest version of the C++ SDK, you can find the associated pod versions on the [`Add Firebase to your project` page](https://firebase.google.com/docs/cpp/setup?platform=ios#libraries-ios).
```

Reply:
Done